src/_pytest/pathlib.py

```diff
@@ -397,3 +402,133 @@ def fnmatch_ex(pattern: str, path) -> bool:
 def parts(s: str) -> Set[str]:
     parts = s.split(sep)
     return {sep.join(parts[: i + 1]) or sep for i in range(len(parts))}
+
+
+class ImportMode(Enum):
+    """Possible values for `mode` parameter of `import_module`"""
+
+    prepend = "prepend"
+    append = "append"
+    importlib = "importlib"
+
+
+class ImportPathMismatchError(ImportError):
+    """Raised on import_module() if there is a mismatch of __file__'s.
+
+    This can happen when `import_module` is given different filenames with the same basename,
+    when those filenames don't reside in packages.
+    """
+
+
+def import_module(
+    p: Union[str, py.path.local, Path],
+    *,
+    mode: Union[str, ImportMode] = ImportMode.prepend
+) -> ModuleType:
+    """
+    Imports and returns a module from the given path.
+
+    The import mechanism used is controlled by the `mode` parameter:
+
+    * `mode == ImportMode.prepend`: the directory containing the module (or package, taking
+      `__init__.py` files into account) will be put at the *start* of `sys.path`, if not
```

Comment:
> if not already in  `sys.path`

According to the code, this is true for `append`, not `prepend`.

Reply:
True, fixed. 👍